lib/awful/widget/calendar_popup.lua

```diff
@@ -246,9 +246,12 @@ end
 --
 -- @param widget Widget to attach the calendar
 -- @tparam[opt="tr"] string position Two characters string defining the position on the screen
+-- @tparam[opt={}] table args Additional options
+-- @tparam[opt=false] bool args.on_hover Show popup while mouse hover
```

Comment:
also, what do you think the fact what click is still binds to the widget? it feels a bit illogical to me -- what both left click and hover are toggling the widget visibility